templates/scenarios/js/sso-tab/README.md

```diff
@@ -84,3 +84,8 @@ Once deployed, you may want to distribute your application to your organization'
 
 - From Visual Studio Code: open the Teams Toolkit and click `Publish to Teams` or open the command palette and select: `Teams: Publish to Teams`.
 - From TeamsFx CLI: run command `teamsfx publish` in your project directory.
+
+## Current limitations on mobile platform
+- On IOS (and a few Android) mobile platform, if user close the Teams App, he would have to sign in again due to browser [blocks third-party cookies](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more) and this results in additional prompts. Related issue can be found [here](https://github.com/OfficeDev/TeamsFx/issues/7574).
+
```

Comment:
IOS -> iOS

Reply:
Or can we say something like: 

This template has adopted [Authorization Code Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) to implement SSO. There's a known limitation with mobile platforms such as iOS and a few Android versions where users have to sign in again as the state was not stored due to [third party cookie being blocked by the browser](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more).

To achieve full SSO experience, we will recommend you to implement OBO flow to pass your token to a server. See more [here](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-code#pass-the-access-token-to-server-side-code)

Reply:
Also I think we can add this as a note after prerequisite section.

Reply:
Sure

Reply:
Updated